src/interfaces/assistants_web/src/components/Shared/Button/Button.stories.tsx

```diff
@@ -73,6 +88,11 @@ export const AllStyles: StoryFn<typeof Button> = () => (
         <Button kind="outline" icon="add" label="Outline" theme="coral" />
         <Button kind="outline" icon="add" label="Outline" theme="quartz" />
         <Button kind="outline" icon="add" label="Outline" theme="mushroom" />
+        <Button kind="outline" icon="add" label="Outline" theme="danger" />
+        <Button kind="outline" icon="add" label="Outline" theme="evolved-blue" />
+        <Button kind="outline" icon="add" label="Outline" theme="evolved-mushroom" />
+        <Button kind="outline" icon="add" label="Outline" theme="evolved-quartz" />
+        <Button kind="outline" icon="add" label="Outline" theme="green" />
```

Comment:
We dont have this themes...

<img width="377" alt="image" src="https://github.com/user-attachments/assets/b5caf486-582d-4e4c-adcf-821607edc843">

Reply:
we need them for assistants custom colors

Reply:
I think we should called them `assistant-accent-{color}`

Reply:
otherwise seems like its part of the theme

Reply:
can we ask them to conform assistants custom colors to the existing themes?

Reply:
lets keep and remove if needed later, but it is more consistent to have all themes for all components